src/Http/Resources/SettingsCollection.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace EscolaLms\Settings\Http\Resources;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+
+class SettingsCollection extends ResourceCollection
+{
+    /**
+     * Transform the resource collection into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        $data = [];
+
+        foreach ($this->collection as $value) {
+            switch($value->type) {
+                case "config":
+                    $data[$value->group][$value->key] = config($value->value);
```

Comment:
I'm not really sure what's the intended use case of "config" type setting?

Reply:
If "config" type settings will be used to overwrite Laravel Config values, then this probably should retrievie `config($value->key)` and not value?

Reply:
true, but `value` cannot be empty. example stripe publish key, smtp email to, etc